tests/test_writer.py

```diff
@@ -1619,7 +1619,6 @@ def test_watermark_rendering(tmp_path):
     assert image_similarity(png_path, target_png_path) >= 0.95
 
 
-@pytest.mark.xfail(reason="issue introduced with pypdf==3.15.4")
 def test_watermarking_reportlab_rendering(tmp_path):
     """
     This test shows that the merged page is rotated+mirrored.
```

Comment:
Shouldn't this comment be updated as well?

Reply:
Yes, that was a bad docstring from my side to begin with. Something like this:

```
"""This test is showing a rotated+mirrored watermark in pypdf==3.15.4."""
```